akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -421,6 +420,7 @@ object Logging {
   private[akka] class LogExt(system: ExtendedActorSystem) extends Extension {
     private val loggerId = new AtomicInteger
     def id() = loggerId.incrementAndGet()
+    def name(clazz: Class[_ <: Actor]): String = "log" + system + "-" + id + "-" + clazz
```

Comment:
perhaps use string interpolation

Reply:
My preference as well, I was sticking with how the other string outputs were in the file. Updated.